energy-growers/src/components/ProductThumbsSection/ProductThumbsSection.scss

```
@@ -0,0 +1,27 @@
+@import "../../assets/styles/globals.scss";
+
+.eg-section-product-thumbs {
+  .eg-thumbs-box-text {
+    display: flex;
+    flex-direction: column;
+    @media screen and (min-width: rem(map-get($breakpoints, sm))) {
+      align-items: center;
+      flex-direction: row;
+      justify-content: space-between;
+    }
+    .eg-thumbs-title {
+      color: map-get($colors, dark-blue);
+    }
+  }
+
+  .eg-container-thumbs-card {
+    padding: rem(33);
+  
+    .eg-container-product-thumbs {
+      
```

```suggestion
       margin-bottom: rem(24);
```

togliamo dal globals e lo diamo qua cosi, se no possiamo rompere tutti i componenti gia sviluppati